src/services/nnas/routes/account-settings.ts

```diff
@@ -0,0 +1,215 @@
+import crypto from 'node:crypto';
+import express from 'express';
+import got from 'got';
+import { z } from 'zod';
+import { getServerByClientID, getPNIDByPID } from '@/database';
+import { LOG_ERROR } from '@/logger';
+import { decryptToken, unpackToken, getValueFromHeaders, sendConfirmationEmail } from '@/util';
+import { config } from '@/config-manager';
+import { HydratedServerDocument } from '@/types/mongoose/server';
+import { HydratedPNIDDocument } from '@/types/mongoose/pnid';
+import { AccountSettings } from '@/types/services/nnas/account-settings';
+import { Token } from '@/types/common/token';
+import { RegionLanguages } from '@/types/services/nnas/region-languages';
+import { RegionTimezone, RegionTimezones } from '@/types/services/nnas/region-timezones';
+import { Country, Region } from '@/types/services/nnas/regions';
+import timezones from '@/services/nnas/timezones.json';
+import regionsList from '@/services/nnas/regions.json';
+
+const router: express.Router = express.Router();
```

Comment:
We can remove this type def. We've started migrating to allowing TypeScript to infer types rather than always set them, unless we need to (reference commit https://github.com/PretendoNetwork/account/commit/cb10a1666812b994f58ebc2fe9a83f03901e6dcc)

This goes for every file here, I'll only comment it once though

```suggestion
const router = express.Router();
```